src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,65 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
+    private static final int FIRST_INDEX = 0;
```

Comment:
you can be more informative - it's clear that it's first from its value, better use in the naming, index of what is it?  like OPERATION_INDEX